### PR TITLE
feat(client): design blog preview

### PR DESCRIPTION
**Pull request by nihalanisumit**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by nihalanisumit** [comment]
preview blog mobile

**Comment by nihalanisumit** [comment]
Screenshot from 2019-08-29 04-16-16
Screenshot from 2019-08-29 04-16-04
Screenshot from 2019-08-29 04-15-47
Screenshot from 2019-08-29 04-15-35
Screenshot from 2019-08-29 04-15-21
Screenshot from 2019-08-29 04-15-13

**Comment by nihalanisumit** [comment]
Screenshot from 2019-08-29 04-15-21
Screenshot from 2019-08-29 04-15-35
Screenshot from 2019-08-29 04-15-47
Screenshot from 2019-08-29 04-16-04
Screenshot from 2019-08-29 04-16-16

File: client/src/components/ViewBlog/ViewBlog.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import UIAvatar from 'react-ui-avatars';
+
```
Comment (piyushagwl007) [inline client/src/components/ViewBlog/ViewBlog.jsx:3]: You could have abstracted the react ui avatars

**Review by piyushagwl007 — APPROVED** [review]

**Pull request merged**